config/environments/test.rb

```diff
@@ -7,7 +7,7 @@
   config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=3600' }
   config.consider_all_requests_local = true
   config.action_controller.perform_caching = false
-  config.cache_store = :null_store
+  config.cache_store = ActiveSupport::Cache.lookup_store(:memory_store)
```

Comment:
```suggestion
  config.cache_store = :memory_store
```
Based on how we do this elsewhere, I think we can simplify:
```
> git grep cache_store
config/environments/development.rb:    config.cache_store = :memory_store
config/environments/development.rb:    config.cache_store = :null_store
config/environments/production.rb:  config.cache_store = :redis_cache_store, { url: IdentityConfig.store.redis_url }
```